chart.js/index.d.ts

```diff
@@ -3,15 +3,15 @@
 // Definitions by: Alberto Nuti <https://github.com/anuti>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare enum ChartType {
+export enum ChartType {
```

Comment:
This may be prior to your changes, but it doesn't look like Chart.js exports or even uses a `ChartType` enum. This may either need to be a `const enum` or be removed.